.pre-commit-config.yaml

```diff
@@ -75,6 +75,41 @@ repos:
   hooks:
   - id: forbid-html-img-without-alt-text
 
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: 'v0.910-1'
+  hooks:
+  - id: mypy
+    additional_dependencies:
+    - types-six
+    - types-backports
+    - types-pyOpenSSL
+    - types-requests
+    args:
+    - --install-types
+    - --non-interactive
+    - --python-version
+    - '3.5'
+    - cheroot/
+    pass_filenames: false
+
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: 'v0.910-1'
+  hooks:
```

Comment:
I just realized that two separate repo entries are unnecessary, just two separate hook entries needed.
```suggestion
```